pkg/proxy/ipvs/proxier.go

```diff
@@ -2160,15 +2156,15 @@ func (proxier *Proxier) cleanLegacyService(activeServices map[string]bool, curre
 			continue
 		}
 		if _, ok := activeServices[cs]; !ok {
-			klog.V(4).Infof("Delete service %s", svc.String())
+			klog.V(4).InfoS("Delete service", "service", svc.String())
 			if err := proxier.ipvs.DeleteVirtualServer(svc); err != nil {
-				klog.Errorf("Failed to delete service %s, error: %v", svc.String(), err)
+				klog.ErrorS(err, "Failed to delete service", "service", svc.String())
 			}
 			addr := svc.Address.String()
 			if _, ok := legacyBindAddrs[addr]; ok {
-				klog.V(4).Infof("Unbinding address %s", addr)
+				klog.V(4).InfoS("Unbinding address", "addr", addr)
 				if err := proxier.netlinkHandle.UnbindAddress(addr, DefaultDummyDevice); err != nil {
-					klog.Errorf("Failed to unbind service addr %s from dummy interface %s: %v", addr, DefaultDummyDevice, err)
+					klog.ErrorS(err, fmt.Sprintf("Failed to unbind service from dummy interface %s", DefaultDummyDevice), "addr", addr)
```

Comment:
Why not remove the fmt.Sprintf and include the DefaultDummyInterface as a structured log field?

Reply:
updated.